src/dygraph-options-reference.js

```diff
@@ -299,6 +299,12 @@ OPTIONS_REFERENCE =  // <JSON>
     "type": "integer",
     "description": "Width, in pixels, of the chart. If the container div has been explicitly sized, this will be ignored."
   },
+  "pixelRatio": {
+    "default": "(devicePixelRatio / context.backingStoreRatio)",
+    "labels": ["Overall display"],
+    "type": "integer",
```

Comment:
Does this have to be an integer? Can it be a float?

Reply:
Yep, good idea, since devicePixelRatio can often be a non-whole number for different zoom levels and certain phones.